build-tests/api-extractor-test-02/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "declarationMap": true,
     "experimentalDecorators": true,
     "strictNullChecks": true,
+    "esModuleInterop": true,
```

Comment:
Your PR only updates the **api-extractor-test-01** project.  Do we need to change the compiler settings for the other packages?  We still need to make sure `esModuleInterop=false` isn't regressed.

Reply:
It seems that api-extractor-test-02 and later use api-extractor-test-01 in some way? If I don't update them as well, the tests crash (meaning, I assume, that they're getting a compiler error).

Reply:
Just circling back here. The current issue is that the tests are tightly coupled, as well as testing that esModuleInterop is assumed false. There are a few ways to decouple them, though you might want to do so in a separate PR. From there you could then have two groups of tests that test both cases.

Reply:
Okay, that sounds reasonable to me.